resources/assets/coffee/react/profile-page/header-extra.coffee

```diff
@@ -97,17 +97,18 @@ class ProfilePage.HeaderExtra extends React.Component
 
           div className: "#{bn}__rows",
             if moment(@props.user.join_date).isBefore moment('2008-01-01')
-              div
-                className: "#{bn}__row"
-                title: moment(@props.user.join_date).format(osu.trans('common.datetime.year_month.moment')),
+              time
```

Comment:
This `time` should be inside the `#{bn}__row` `div` instead of replacing it, otherwise the alignment of the tooltip ends up being off:

![](https://puu.sh/zoKKk/1d4ec7fe70.png)